VideoApi/VideoApi/Controllers/ConferenceController.cs

```diff
@@ -478,6 +480,44 @@ public async Task<IActionResult> GetHostsInHearingsTodayAsync()
 
             return Ok(response);
         }
+        
+        /// <summary>
+        /// Get conferences Hearing rooms
+        /// </summary>
+        /// <returns>Hearing rooms details</returns>
+        [AllowAnonymous]
+        [HttpGet("dateStamp/hearingRooms")]
+        [OpenApiOperation("GetConferencesHearingRooms")]
+        [ProducesResponseType(typeof(List<ConferenceHearingRoomsResponse>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(string), (int) HttpStatusCode.NoContent)]
+        public async Task<IActionResult> GetConferencesHearingRoomsAsync([FromQuery]string dateStamp)
+        {
+            _logger.LogDebug("GetConferencesHearingRooms");
+
+            try
+            {
+                var date = DateTime.ParseExact(dateStamp, "yyyy-MM-dd", CultureInfo.InvariantCulture);
+
+                var interpreteRooms = await _queryHandler.Handle<GetConferenceInterpreterRoomsByDateQuery, List<HearingAudioRoom>>(
```

Comment:
Small misspelling here change to interpreterRooms
```suggestion
                var interpreterRooms = await _queryHandler.Handle<GetConferenceInterpreterRoomsByDateQuery, List<HearingAudioRoom>>(
```

Reply:
fixed it. thanks